README.md

```diff
@@ -2,9 +2,72 @@
 
 ABDM integrator is a REST API service packaged as a django app. 
 It implements all the three milestones of [ABDM](https://sandbox.abdm.gov.in/abdm-docs/getting-started).
+This package is intended to be used by developers aiming to perform integration with ABDM using python.
 
 ## Requirements
-
 - python >= 3.8
 - django >= 3.2
 - djangorestframework >= 3.12
+- requests
+
+## Build from source
+
+1. Clone the repository.
+2. Install python build. (If not already existing)
+    ```commandline
+     pip install build
+    ```
+3. Run below command to build.
+    ```commandline
+    python -m build
+    ```
+The resulting package (wheel or tar) can be then installed using pip into your Django Project.
+
+## Settings
+1. Add `abdm_python_integrator` to Django `INSTALLED_APPS` setting.
+2. Include app urls into your Django root url config.
+    ```
+    url(r'^abdm/', include('abdm_python_integrator.urls')),
+    ```
+3. Add below settings to your Django settings file.
+
+    ```python
+    ABDM_INTEGRATOR = {
+        # Client ID provided by ABDM. 
+        # Required setting.
+        'CLIENT_ID': 'dummy-value',
```

Comment:
(Assuming you meant test_settings.py)
I do feel like having this here makes it convenient for a user to get started. 
One minor difference is the test_settings uses `''` as urls (as an indication to avoid network calls) and Readme uses examples from the sample environment.

Reply:
Sure, may be they can be duplicated into settings.